### PR TITLE
Verify and standardize all conda configs to show channels

### PR DESCRIPTION
**Pull request by jakirkham**
Related: https://github.com/conda-forge/conda-smithy/pull/137

Just make sure we are getting the channel listings as early as possible.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -72,10 +72,10 @@ install:
           else
             {$root = "C:\Miniconda-x64"}
           $env:path="$root;$root\Scripts;$root\Library\bin;$($env:path)"
+    - cmd: conda config --set show_channel_urls true
     - cmd: conda update --yes --quiet conda
     - cmd: set PYTHONUNBUFFERED=1
 
-    - cmd: conda config --set show_channel_urls yes
```
Comment (jakirkham) [inline appveyor.yml]: Turns out this wasn't set early enough to truly show everything.

Comment (ocefpaf) [inline appveyor.yml]: And we never noticed :scream:

**Comment by jakirkham** [comment]
Would you mind take a look, @ocefpaf?

File: appveyor.yml (hunk 1 above)
Comment (jakirkham) [inline appveyor.yml]: Yeah, it's not a crisis as `defaults` is basically the only one. However, if we start wanting to use a newer version of jinja for instance, we might need to change that.

Comment (jakirkham) [inline appveyor.yml]: I was inspired to look based on [your change in `gdal`]( https://github.com/conda-forge/gdal-feedstock/pull/37 ).

**Comment by ocefpaf** [comment]
LGTM

File: appveyor.yml (hunk 1 above)
Comment (ocefpaf) [inline appveyor.yml]: > I was inspired to look based on [your change in `gdal`]( https://github.com/conda-forge/gdal-feedstock/pull/37 ).

I will removed that BTW. I noticed the channels were not listed and I had no idea if the dependencies were coming from `defaults` or `conda-forge`.

Comment (jakirkham) [inline appveyor.yml]: Sure, I understand. It is useful for debugging. So we really should have that info anywhere we can as you suggested. Hence these PRs.

**Pull request merged**